src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -240,6 +242,41 @@ class spell_warr_charge_effect : public SpellScriptLoader
         }
 };
 
+// 167105 - Colossus Smash 7.1.5
+class spell_warr_colossus_smash : public SpellScriptLoader
+{
+public:
+    spell_warr_colossus_smash() : SpellScriptLoader("spell_warr_colossus_smash") { }
+
+    class spell_warr_colossus_smash_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_warr_colossus_smash_SpellScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(SPELL_WARRIOR_COLOSSUS_SMASH_EFFECT))
```

Comment:
I thing is good idea to use the new ValidateSpellInfo function.

```
return ValidateSpellInfo
({
    SPELL_WARRIOR_COLOSSUS_SMASH_EFFECT
});
```

Reply:
it's meant to be in one line tho